modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/type/facet/view.jsp

```diff
@@ -19,85 +19,118 @@
 <%@ taglib uri="http://java.sun.com/portlet_2_0" prefix="portlet" %>
 
 <%@ taglib uri="http://liferay.com/tld/aui" prefix="aui" %><%@
+taglib uri="http://liferay.com/tld/ddm" prefix="liferay-ddm" %><%@
 taglib uri="http://liferay.com/tld/ui" prefix="liferay-ui" %>
 
 <%@ page import="com.liferay.petra.string.StringPool" %><%@
 page import="com.liferay.portal.kernel.util.HtmlUtil" %><%@
 page import="com.liferay.portal.kernel.util.WebKeys" %><%@
 page import="com.liferay.portal.search.web.internal.facet.display.context.AssetEntriesSearchFacetDisplayContext" %><%@
-page import="com.liferay.portal.search.web.internal.facet.display.context.AssetEntriesSearchFacetTermDisplayContext" %>
+page import="com.liferay.portal.search.web.internal.facet.display.context.AssetEntriesSearchFacetTermDisplayContext" %><%@
+page import="com.liferay.portal.search.web.internal.type.facet.configuration.TypeFacetPortletInstanceConfiguration" %>
+
+<%@ page import="java.util.HashMap" %><%@
+page import="java.util.List" %><%@
+page import="java.util.Map" %>
 
 <portlet:defineObjects />
 
 <%
 AssetEntriesSearchFacetDisplayContext assetEntriesSearchFacetDisplayContext = (AssetEntriesSearchFacetDisplayContext)java.util.Objects.requireNonNull(request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT));
+
+if (assetEntriesSearchFacetDisplayContext.isRenderNothing()) {
+	return;
+}
+
+TypeFacetPortletInstanceConfiguration typeFacetPortletInstanceConfiguration = assetEntriesSearchFacetDisplayContext.getTypeFacetPortletInstanceConfiguration();
+
+Map<String, Object> contextObjects = new HashMap<String, Object>();
+
+contextObjects.put("namespace", renderResponse.getNamespace());
+contextObjects.put("assetEntriesSearchFacetDisplayContext", assetEntriesSearchFacetDisplayContext);
+
+List<AssetEntriesSearchFacetTermDisplayContext> assetEntriesSearchFacetTermDisplayContexts = assetEntriesSearchFacetDisplayContext.getTermDisplayContexts();
 %>
 
 <c:choose>
 	<c:when test="<%= assetEntriesSearchFacetDisplayContext.isRenderNothing() %>">
 		<aui:input autocomplete="off" name="<%= HtmlUtil.escapeAttribute(assetEntriesSearchFacetDisplayContext.getParameterName()) %>" type="hidden" value="<%= assetEntriesSearchFacetDisplayContext.getParameterValue() %>" />
 	</c:when>
 	<c:otherwise>
-		<liferay-ui:panel-container
-			extended="<%= true %>"
-			id='<%= renderResponse.getNamespace() + "facetAssetEntriesPanelContainer" %>'
-			markupView="lexicon"
-			persistState="<%= true %>"
-		>
-			<liferay-ui:panel
-				collapsible="<%= true %>"
-				cssClass="search-facet"
-				id='<%= renderResponse.getNamespace() + "facetAssetEntriesPanel" %>'
-				markupView="lexicon"
-				persistState="<%= true %>"
-				title="type"
+		<aui:form method="post" name="assetEntriesFacetForm">
```

Comment:
Hey @oliv-yu,

For the name of the form, we typically name them `name="fm"` or `name="dialogFm"`.  Can you update and follow a similar pattern?

Please let me know if you have any questions. Thanks!